cmd/chain-maind/app/app.go

```diff
@@ -38,7 +39,6 @@ import (
 	"github.com/crypto-org-chain/chain-main/v1/app/params"
 	"github.com/crypto-org-chain/chain-main/v1/config"
 	chainmaincli "github.com/crypto-org-chain/chain-main/v1/x/chainmain/client/cli"
-	"github.com/google/renameio"
```

Comment:
maybe also remove the dependency from go.mod?

Reply:
ok, i'll check